fil_actor_interface/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "fil_actor_interface"
 description = "Backward compatibility layer for all versions of builtin actor released v8 onwards."
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/ChainSafe/fil-actor-states"
-version = "5.1.0"
+version = "5.2.0"
```

Comment:
shouldn't we do it in a separate release PR? There was some automatic release workflow which would trigger only on release PRs (properly labelled). 

https://github.com/ChainSafe/fil-actor-states/blob/main/.github/workflows/tests.yml#L48-L49

Reply:
cc @sudo-shashank what do you think?

Reply:
yes i prefer a separate release PR with version bump, also we need to have a  `Release` label attached to the PR, which enables additional forest test and calibnet sync check to make sure everything is good.
For publishing all we need to do is create and push a new version tag

Reply:
version bump reverted, @sudo-shashank could u help with a new release?